test-util.sh

```diff
@@ -24,6 +24,12 @@ function fail() {
 }
 
 function check() {
+  local this_test="$1"
+  shift
+  check_pipeline "$this_test" "$INVOCATION $@"
```

Comment:
This logic isn't super obvious: peeling off `this_test` then adding it back to the `check_pipeline` call seems to allow us ot interject `$INVOCATION`. But for test-siz2fastq.sh that's empty. `test-align.sh` has an invocation, but presumably we could use the same string trick as in `test-siz2fastq.sh` to execute multipart commands?

I assume not, and this is how it must be, but yeah: not obvious.

Reply:
We could use the same string trick, yes, but `test-align.sh` is much clearer using bare args without the trick.